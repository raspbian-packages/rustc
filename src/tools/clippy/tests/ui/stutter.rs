#![feature(plugin)]
#![plugin(clippy)]
#![warn(stutter)]
#![allow(dead_code)]

mod foo {
    pub fn foo() {}
    pub fn foo_bar() {}
    pub fn bar_foo() {}
    pub struct FooCake {}
    pub enum CakeFoo {}
}

fn main() {}
