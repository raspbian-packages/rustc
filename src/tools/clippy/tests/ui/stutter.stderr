error: item name starts with its containing module's name
 --> $DIR/stutter.rs:8:5
  |
8 |     pub fn foo_bar() {}
  |     ^^^^^^^^^^^^^^^^^^^
  |
  = note: `-D stutter` implied by `-D warnings`

error: item name ends with its containing module's name
 --> $DIR/stutter.rs:9:5
  |
9 |     pub fn bar_foo() {}
  |     ^^^^^^^^^^^^^^^^^^^

error: item name starts with its containing module's name
  --> $DIR/stutter.rs:10:5
   |
10 |     pub struct FooCake {}
   |     ^^^^^^^^^^^^^^^^^^^^^

error: item name ends with its containing module's name
  --> $DIR/stutter.rs:11:5
   |
11 |     pub enum CakeFoo {}
   |     ^^^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

